docs/advanced_syntax/builtins.md

```diff
@@ -0,0 +1,50 @@
+# Builtins
+
+Builtins are native methods (built in the compiler itself) reserved keywords in Amber.  
+
+Similar to the standard library, they generate valid [Shellcheck](https://www.shellcheck.net/) code (though full support for this in Amber is still in progress).
+
+## Cd
+
+A classic command in Bash scripts is `cd` which changes the current directory, require a `Text` parameter.
```

Comment:
The sentence talks about the command as if someone describes it, therefore it should be "requires" not "require".